src/main/java/actions/LocomotiveAction.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2019 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package actions;
```

Comment:
You might want to move everything into `org.terasology.actions` and such - there's a quirky situation where that's required for some things (reflections?). Probably doesn't matter here but until we clear that up all module packages should be under `org.terasology.something`